qutebrowser/browser/commands.py

```diff
@@ -870,37 +870,44 @@ def _yank_to_target(self, s, sel, what, keep):
 
     @cmdutils.register(instance='command-dispatcher', scope='window')
     @cmdutils.argument('count', count=True)
-    def zoom_in(self, count=1):
+    @cmdutils.argument('quiet', flag='q')
```

Comment:
This isn't needed, the first letter is used by default for the flag.

(same for all three changes)